NewFeature

```diff
@@ -0,0 +1,8 @@
+NewFeature
+Adf
+Asdfggf
+Sdfg
+Fix fix fix
+Asdsdf
```

Comment:
I thnk this name is not clear. Can you create a better one?

Reply:
no, thanks :)

Reply:
Ok, fine